docs/resources/client-start-help

```diff
@@ -25,10 +25,11 @@ Flags:
       --metrics.networkMetricsTick duration        Network metrics check tick in seconds. (default 1m0s)
       --metrics.ethereumMetricsTick duration       Ethereum metrics check tick in seconds. (default 10m0s)
       --diagnostics.port int                       Diagnostics HTTP server listening port. (default 8081)
-      --tbtc.preParamsPoolSize int                 tECDSA pre-parameters pool size. (default 50)
+      --tbtc.preParamsPoolSize int                 tECDSA pre-parameters pool size. (default 3000)
       --tbtc.preParamsGenerationTimeout duration   tECDSA pre-parameters generation timeout. (default 2m0s)
       --tbtc.preParamsGenerationDelay duration     tECDSA pre-parameters generation delay. (default 10s)
       --tbtc.preParamsGenerationConcurrency int    tECDSA pre-parameters generation concurrency. (default 1)
+      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default 10)
```

Comment:
I think the default here is the number of cores.

Reply:
Yeah. I updated it in https://github.com/keep-network/keep-core/pull/3207/commits/003c8ef28ff5c83e2358e44cb1d0bcea7ad768f4. The problem with this file though is that it's auto-generated when running `make`. And this value will go back to 10 if someone regenerates it on a 10-core machine.

I do not want to solve it now and block this work but that's something we'll have to look at in the future cc @nkuba